fixed iptables disabled recipe

hagay3 opened the pull request:
### Description

flush (deletes) iptables rules

### Issues Resolved

disable recipe not deletes old rules

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

tas50 commented:
This will run on every Chef run. You'll probably want to use a notifies from the service stop to ensure this dosn't do that. Also you'll need to sign this commit for DCO before we can merge it.

hagay3 commented:
You are right, added the notifies thing. 
I cant add it to the service stop because it has "only_if" statement. 
So it runs only on rhel

I added it to the folder deletion.

tas50 commented:
Looks good. Thanks
